src/components/Button/index.js

```diff
@@ -1,8 +1,14 @@
 import React from "react";
+import "./button-add.css";
 
-export default function Button({ onClick, label }) {
+export default function Button({ onClick, label, inputType }) {
   return (
-    <button style={{ margin: "1rem" }} onClick={onClick}>
+    <button
+      style={{ margin: "1rem" }}
+      onClick={onClick}
+      type={inputType === 1 ? "submit" : "button"}
```

Comment:
Why input type is a number instead of a string?

Reply:
Also, declare a const with the exact logic you have to calculate the type and choose a good name for it